modules/gitops-argo-rollouts-cli-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI overview
+
+You can use the Argo Rollouts CLI, which is an optional plugin, to manage and monitor Argo Rollouts resources directly, bypassing the need to use the {OCP} web console or the {oc-first}.
```

Comment:
`oc-first` isn't rendering in the preview. Verify this before merge.

Reply:
Yeah, this attribute is not merging because it is not defined in the attributes file used in GitOps. Can we stick to {`oc`}???

Reply:
Keeping it as {oc} until advised otherwise.